doc/user-guide/06-layout-configuration.adoc

```diff
@@ -143,61 +106,60 @@ we mount the +/dev/mapper/backup-backup+ filesystem on
 +/media/backup+, it will get excluded, as the mountpoint is under
 +/media+.
 
-=== Manual excludes
-It seems prudent to prevent the external drives from ever being backed-up or overwritten. The default configuration contains these lines:
+If we mount the filesystem on +/dev/mapper/backup-backup+ on +/backup+,
+Relax-and-Recover will think that it's necessary to recreate the filesystem:
```

Comment:
Could you rephrase it so that it does no longer look
as if Relax-and-Recover has the ability to think
e.g. like
```
If we mount the filesystem on /dev/mapper/backup-backup on /backup,
Relax-and-Recover will recreate this filesystem
Relax-and-Recover recreates mounted filesystems
except filesystems that get excluded by default.
```
This is only an offhanded proposal.
I did not verify that ReaR really recreates filesystems
only depending on whether or not they are mounted.

Reply:
> I did not verify that ReaR really recreates filesystems
> only depending on whether or not they are mounted.

I verified that, actually.

Reply:
See also https://github.com/rear/rear/issues/3129